src/things.ts

```diff
@@ -0,0 +1,724 @@
+import Errors from './errors'
```

Comment:
why do you have this file in this pr if it's just going to be having commented code. If it's being fixed in another pr remove it from this one.

Reply:
Some users.ts functions need the things.ts interfaces and url so I added it. But I will remove the commented out code